src/foam/u2/view/MultiChoiceView.js

```diff
@@ -167,20 +167,45 @@ foam.CLASS({
       return this.dao.where(this.IN(of.ID, this.data));
     },
 
+    function isChoiceSelected(data, choice){
```

Comment:
This seems easier to understand:

function isChoiceSelected(data, choice) {
      for ( var i = 0 ; i < data.length ; i++ )
        if ( foam.util.equals(data[i], choice) ) return true;

      return false;
    },

Reply:
RE: comment below